assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -312,6 +338,71 @@ const SelectedVehicleDataLayers = ({
   )
 }
 
+const SelectedRouteDataLayers = ({
+  routePatternIdentifier,
+  routePatterns,
+  showSelectionCard,
+  deleteSelection,
+  selectVehicle,
+  selectRoutePattern,
+  setStateClasses,
+}: {
+  routePatternIdentifier: RoutePatternIdentifier
+  routePatterns: ByRoutePatternId<RoutePattern> | null
+  showSelectionCard: boolean
+  selectVehicle: (vehicleOrGhost: VehicleOrGhost) => void
+  selectRoutePattern: (routePattern: RoutePattern) => void
+  deleteSelection: () => void
+
+  setStateClasses: (classes: string | undefined) => void
+}) => {
+  const selectedRoutePattern: RoutePattern | undefined = routePatterns
+    ? routePatterns[routePatternIdentifier.routePatternId]
+    : undefined
+  const position = selectedRoutePattern
+    ? selectedRoutePattern.shape?.points?.map((p) =>
+        Leaflet.latLng(p.lat, p.lon)
+      ) || []
+    : []
```

Comment:
**nitpick:** it would probably be slightly clearer to name this `positions` since it's a list.